plugin/trino-hive/src/main/java/io/trino/plugin/hive/metastore/HiveMetastoreModule.java

```diff
@@ -45,7 +45,9 @@ protected void setup(Binder binder)
             bindMetastoreModule("thrift", new ThriftMetastoreModule());
             bindMetastoreModule("file", new FileMetastoreModule());
             bindMetastoreModule("glue", new GlueMetastoreModule());
-            bindMetastoreModule("alluxio", new AlluxioMetastoreModule());
+            // Alluxio is not maintained and accumulates security vulnerabilities,
+            // so we want to avoid loading any classes for it if it is not used
+            bindMetastoreModule("alluxio", deferredModule("io.trino.plugin.hive.metastore.alluxio.AlluxioMetastoreModule"));
```

Comment:
do we have any testing for alluxio? Maybe `TestHivePlugin`?

Reply:
There's `TestHiveAlluxioMetastore` but it mostly disables tests because some feature is not supported. It can't be run locally, it seems, because it requires some externally-provided properties.

Reply:
...and it instantiates `AlluxioHiveMetastore` directly

Reply:
I added a simple test to `TestHivePlugin`, thanks for the suggestion